src/server/game/Spells/SpellEffects.cpp

```diff
@@ -6364,6 +6364,10 @@ void Spell::EffectPlaySound(SpellEffIndex effIndex)
         return;
     }
 
+    Player* player = unitTarget->ToPlayer();
```

Comment:
Remove this one, as this check is already here https://github.com/azerothcore/azerothcore-wotlk/blob/c98eceba22acb58c7b90685a21c9bf2781d1826e/src/server/game/Spells/SpellEffects.cpp#L6361
It causes a redefinition compilation error.